jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -30,10 +30,6 @@ fi
 [[ -z ${NEED_C17_COMPILER} ]] && NEED_C17_COMPILER=false
 
 export APT_PARAMS=
-# workaround for changing our packages testing server
-if [[ ${DISTRO} != 'xenial' && ${DISTRO} != 'trusty' && ${DISTRO} != 'stretch' ]]; then
-  export APT_PARAMS="--allow-releaseinfo-change"
```

Comment:
should this line stay, since the logic checks if the `DISTRO` is not `xenial` or `trusty` or `stretch`?

Reply:
If I remember correctly my motivation was more about removing the workaround of `allow-releaseinfo-change` that was in place during the transition to the new packages server. It should not be needed anymore.